mythril/analysis/modules/ether_thief.py

```diff
@@ -120,6 +120,8 @@ def _analyze_state(state):
             state.current_transaction.caller == ATTACKER_ADDRESS,
         ]
 
+        log.debug("Constraints: %s", constraints)
```

Comment:
Printing the constraints in the debug information can get overly verbose, even in `debug`. Not only that, but even if the debug is not printed, this can sometimes cause a slowdown as well.

Reply:
Slowdowns are mitigated by the fact that the string is only generated if printed (with passing constraints as a supplemental argument).
But ok, let me remove it.